src/openrct2/scenario/Scenario.h

```diff
@@ -366,6 +366,8 @@ enum
 #define AUTOSAVE_PAUSE 0
 #define DEFAULT_NUM_AUTOSAVES_TO_KEEP 10
 
+static constexpr money32 INVALID_VALUE_ON_SCENARIO_FAILURE = 0x80000001;
```

Comment:
```suggestion
static constexpr money32 COMPANY_VALUE_FAILED_OBJECTIVE = 0x80000001;
```

Reply:
Thanks for the suggestion, I actually went with `COMPANY_VALUE_ON_FAILED_OBJECTIVE`, let me know if you disagree.